packages/meter/README.md

```diff
@@ -63,16 +63,38 @@ import { Meter } from '@spectrum-web-components/meter';
 
 ## Variants
 
-### Over Background
+### Informative
 
-When a loader needs to be placed on top of a colored background, use the over background loader as signified by `[over-background]`. This loader uses a white opaque color no matter the background. Make sure the background offers enough contrast for the loader to be legible.
+By default, the informative variant can be used to represent a neutral or non-semantic value, such as the number of tutorials completed.
 
 ```html
-<div
-    style="width: 240px; height: 160px; display: flex; flex-direction: column; align-items: center; justify-content: space-around; background-color: var(--spectrum-transparent-black-400);"
->
-    <sp-meter progress="42" over-background>Tasks Completed</sp-meter>
-</div>
+<sp-meter progress="50">Storage Space</sp-meter>
+```
+
+### Positive
```

Comment:
Hrm. If we're making a breaking change _anyways_ should we discuss flattening all of these booleans into a `variant` property like we see in `<sp-button>` or similar?

Reply:
@Westbrook yea that sounds great! 
1. new variants `positive, notice, negative` has been added and can be customized as 
`<sp-meter variant="positive">` instead of `<sp-meter positive>`
2. Tests added
3. Storybook updated